src/utils.tsx

```diff
@@ -393,3 +398,25 @@ export const ConditionalWrapper = ({
     wrapper,
     children,
 }: WrapperProps) => (condition ? wrapper(children) : children)
+
+export function getTranslation(
+    translationObject: TranslatedString,
+    languageId = 'nb',
+): string | null {
+    const translations: Translation[] = translationObject.translation
+    const match = translations.find(
+        (currentTranslation) => currentTranslation.language === languageId,
+    )
+    if (!match) return null
+    return match.value
+}
+
+export function includesStation(
+    stations: Station[],
+    searchElement: Station,
+): boolean {
+    for (let index = 0; index < stations.length; index++) {
```

Comment:
Heile løkka kan erstattast med `.some`:
```
return stations.some(station => station.id === searchElement.id)
```
og då er det kanskje unødvendig med eigen util for dette?

Reply:
Bra tips! Nei, blir vel strengt tatt litt overflødig med en util da:) Refaktorerer denne.